examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -64,9 +73,30 @@
 goes_ts = ts.TimeSeries(goes_data, meta, units, source="xrs")
 
 ###############################################################################
-# Finally, we can plot the timeseries.
+# NOAA also provides the past 7 days of flare information which we can also parse.
+
+flare_events = Table.read("https://services.swpc.noaa.gov/json/goes/primary/xray-flares-7-day.json", format='pandas.json')
+
+###############################################################################
+# Finally, we can plot the timeseries and overlay all the flares that occurred.
+# We will do this only for the last day otherwise the plot may be too busy.
 
 fig, ax = plt.subplots()
 goes_ts.plot(axes=ax)
```

Comment:
```suggestion
goes_ts.plot(axes=ax)
plot_start = goes_ts.time[-1] - 1 * u.day
ax.set_xlim(
    plot_start.datetime, (plot_start + 1 * u.day).datetime
)
```
More readable to define and use a variable `plot_start`

Reply:
I do have a question for you: why did you make this plot go backwards in time?  Is that a common way to visualize this data?

Reply:
I removed this.